src/system_identity/output_platform.rs

```diff
@@ -0,0 +1,8 @@
+use std::path::PathBuf;
+
+// TODO: Currently this is the only supported output for the application, so we are
+// not actually using this enum yet.
+pub enum AuthOutputPlatform {
+    LocalPrivateKeyPath(PathBuf),
+    // Vault(VaultConfig), // TODO: Vault
+}
```

Comment:
Nit: instead of a `TODO` I'd explain that we keep the enum even if we don't need it yet because we will need to support additional outputs in a near future.

Reply:
From another conversation with @rubenruizdegauna I think I'll actually use the enum 😆